panel/scripts/docs.js

```diff
@@ -0,0 +1,24 @@
+
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
+const docgen = require("vue-docgen-api");
+ 
+glob("src/components/**/*.vue", async (err, files) => {
+  if (err) {
+    return console.log(err);
+  }
+
+  const parsing = files.map(async file => {
+    let data = await docgen.parse(file, {
+      alias: { '@': path.resolve(__dirname, '../src/') },
+    });
+    data.srcFile = file;
+    return data;
+  });
+
+  const components = await Promise.all(parsing);
+  fs.writeFileSync("dist/components.json", JSON.stringify(components));
```

Comment:
I think this should use an absolute path based on `__dirname` as well to avoid strange bugs down the line when the file is updated in the wrong place.